gravitee-apim-rest-api/postman/scripts/tests.sh

```diff
@@ -47,9 +47,9 @@ echo "-------------------------------------------------"
 echo ""
 
 # Check if POSTMAN_ENV value is a valid environment
-if [ $POSTMAN_ENV != "demo" ] && [ $POSTMAN_ENV != "nightly" ] && [ $POSTMAN_ENV != "localhost" ]; then
+if [ $POSTMAN_ENV != "localhost" ]; then
```

Comment:
if localhost is the only available value, why keeping the possibility to change it ?
We could just have "localhost" hardcoded.